docs/repositories-configure/managing-branches.md

```diff
@@ -3,6 +3,9 @@
 {% if not extra.self_hosted %}
 Codacy automatically analyzes the default branch of your repository (typically `master` or `main` as configured on your Git provider), and also supports analyzing multiple branches.
 
+!!! info
+    Codacy displays information for the default branch on dashboards. Consider matching the default branch on Codacy to the default branch on your Git provider.
```

Comment:
Just two comments regarding this information:

- In the current version, users may understand that Codacy **only** displays information on dashboards for the default branch. We're not saying that Codacy displays those branches by default (implying that users can then change the branch). It becomes a bit tricky to add this information because of the repetition "By default, Codacy displays the default branch". :sweat_smile:
- I'd experiment with having this information directly as a paragraph (ditching the admonition). After all, it's the explanation of what a default branch is on the Codacy side.

Reply:
Yeah, it's a tricky sentence! I'll let it simmer until tomorrow.

Reply:
Following up on https://github.com/codacy/docs/pull/1576/commits/82bd26de66833ee8fa32b5ed3772f477ded3b23b